automatic/firefox/tools/chocolateyInstall.ps1

```diff
@@ -48,3 +89,24 @@ if ($alreadyInstalled -and ($env:ChocolateyForce -ne $true)) {
 
   Install-ChocolateyPackage @packageArgs
 }
+
+if ($PackageParameters['InstallDir']) {
+  $installPath = $PackageParameters['InstallDir']
+}
+else {
+  $installPath = "C:\Program Files\Mozilla Firefox"
```

Comment:
@majkinetor do you know of a way to get the default installation path? This will probably break for 32 bit installs.

Reply:
anyone?

Reply:
Why do you need it ? Can you leave it empty and use `Get-AppInstallLocation` from core extension afterwards to get it when user didn't provide it (if you need it later for something)

Reply:
I need to know where to put the `policies.json` files which is in a subdirectory `distribution` in the location of the exe. That's exactly the command I was looking for. Thanks, will use that.

Reply:
There is also more legit option, to use Registry Uninstaller stuff. From my  machine:

```

Registry key: HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall\Mozilla Firefox 72.0.1 (x64 en-US)
DisplayName   Mozilla Firefox 72.0.1 (x64 en-US)  
DisplayVersion   72.0.1  
Publisher   Mozilla  
InstallLocation   C:\Program Files\Mozilla Firefox  
UninstallString   "C:\Program Files\Mozilla Firefox\uninstall\helper.exe"  
DisplayIcon   C:\Program Files\Mozilla Firefox\firefox.exe,0  
HelpLink   {https://support.mozilla.org}  
URLInfoAbout   {https://www.mozilla.org}  
URLUpdateInfo   {https://www.mozilla.org/firefox/72.0.1/releasenotes}  
```